toptal/picasso
# feat: convert Form to non-compound component

[FX-3590](https://toptal-core.atlassian.net/browse/FX-3590)

### Description

The PR adds exported members for all Form components in Picasso Forms to be available directly from entry point as non-compound components.

note: the root picasso package does not require a changeset, only has imports updated in a story

### How to test

- See if all the components are exported properly
- Try the codemod on any codebase that uses compound components `npx @toptal/picasso-codemod@5.3.2-alpha-fx-3590-compound-components-are-increasing-bundle-size-c57a541d.467 v52.2.0/non-compound-forms src/**/*.tsx`
- Verify codemod produces working code and no leftover compound imports are left
- See if there are any remaining examples / documentation still recommending to use compound components


### Development checks



> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[FX-3590]: https://toptal-core.atlassian.net/browse/FX-3590?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
